osu.Game/Graphics/Containers/SectionsContainer.cs

```diff
@@ -240,7 +240,9 @@ protected override void UpdateAfterChildren()
                 headerBackgroundContainer.Height = expandableHeaderSize + fixedHeaderSize;
                 headerBackgroundContainer.Y = ExpandableHeader?.Y ?? 0;
 
-                float smallestSectionHeight = Children.Count > 0 ? Children.Min(d => d.Height) : 0;
+                var flowChildren = scrollContentContainer.FlowingChildren.OfType<T>();
```

Comment:
`flowChildren` is enumerated thrice in this method. The underlying `FlowingChildren` includes two live order-by calls, so I *believe* this may end up re-sorting every time?

Maybe worth materialising this collection here?

Reply:
I saw this but the overhead didn't *seem* too bad (could not notice on frame graph). Likely due to number of sections being very limited in all cases.